src/System/Hapistrano.hs

```diff
@@ -330,6 +331,13 @@ restartServerCommand = do
     Nothing -> return "No command given for restart action."
     Just cmd -> runCommand (host conf) cmd
 
+cleanBuildScript :: [String] -> [String]
+cleanBuildScript allScriptLines = filter (not . isCommentOrEmpty) allScriptLines
+  where
+    isCommentOrEmpty line = isEmpty line || isComment line
```

Comment:
Indeed,  It trims these chars `\t, \n, \r, \f, \v` as defined by is the function [isSpace](https://hackage.haskell.org/package/base-4.9.0.0/docs/Data-Char.html#v:isSpace).
I think using a parsec might be an overkill since hapistrano is pretty lightweight in dependencies.

\cc: @jpvillaisaza any thoughts on this?